src/qmapshack/canvas/CCanvas.h

```diff
@@ -100,6 +100,7 @@ class CCanvas : public QWidget
     void zoomTo(const QRectF& rect);
     void displayInfo(const QPoint& px);
     poi_t findPOICloseBy(const QPoint& px) const;
+    QList<poi_t> findPoisIn(const QRectF degRect) const;
```

Comment:
`const QRectF degRect`

Reply:
we are not doing Python here where everything is a reference.

Reply:
Yes, sorry :facepalm: